scripts/qemu/run-tests.sh

```diff
@@ -20,6 +32,15 @@ ctest --verbose
 numactl -N 0 ctest --output-on-failure
 numactl -N 1 ctest --output-on-failure
 
+if [ "$COVERAGE" = "COVERAGE" ]; then
+	COVERAGE_FILE_NAME=exports-coverage-qemu-$CONFIG_NAME
+	echo "COVERAGE_FILE_NAME: $COVERAGE_FILE_NAME"
+	../scripts/coverage/coverage_capture.sh $COVERAGE_FILE_NAME
+	mkdir -p $COVERAGE_DIR
```

Comment:
this line is duplicated (it's created on top of the file)

Reply:
Removed.

Done.